src/utils/solanaUtils.ts

```diff
@@ -959,3 +974,116 @@ export const getAllTransactions = async (
     return []
   }
 }
+
+/**
+ *
+ * @param connection Solana connection
```

Comment:
This says connection but param is cluster

Reply:
Same with line 1045

Reply:
Fixed